app/src/pages/inside/profilePage/localizationBlock/localizationBlock.jsx

```diff
@@ -33,7 +33,7 @@ import {
 } from 'common/constants/supportedLanguages';
 import { langSelector, setLangAction } from 'controllers/lang';
 import { InputDropdown } from 'components/inputs/inputDropdown';
-import { PROFILE_PAGE_EVENTS } from 'components/main/analytics/events';
+import { PROFILE_EVENT } from 'components/main/analytics/events/ga4Events/profilePageEvent';
```

Comment:
We have an alias for that path. You can find it in basic webpack config

Reply:
fixed ✅